### ISSUE TITLE
[CAIA a11y Research - PILOT]: VHA Payment History/One thing per page - Mobile Pilot

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#93594]
> [!IMPORTANT]
> - **TICKET STATUS:** `Completed`
> - **PILOT DATE:**  9/26/24, 4:30pm ET
> - **TYPES OF ASSISTIVE TECH:**  **Mobile Testing**: iOS VoiceOver with Safari
> - **TESTING ENVIRONMENT:** [Codepen](https://codepen.io/team/Debt-Team/live/rNXBLoZ)

# CAIA A11y Support
This is a child ticket of the research ticket [93530](https://github.com/department-of-veterans-affairs/va.gov-team/issues/93530) - created to track work, completed from this pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- **Pilot**: Maria
- **Co-pilot**: Jamie

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- **Team name:** `Debt Resolution`
- **OCTO product owner:** `Denise Coveyduc`
- **Product name:** `Debt Portal`
- **Product manager:** `Tom Davis`
- **Slack channel:** `[#debt-resolution](https://join.slack.com/share/enQtNzM5MTE3ODIwMDQ4My1hYTY3OGIyZGYzMGYwZjkyYWYyMzg5OTE4MDhiODRiOTU5ZGQ3ZGIzOTMyNTRjMzcwMjA4ZGZkZTE5MjNjY2Fm)`
- **Dedicated content writer on your team (if you have one):** `writer`
- **Dedicated a11y specialist on your team (if you have one):** `a11y`

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** `9/26/24`
- **Prototype**: [Codepen](https://codepen.io/team/Debt-Team/live/rNXBLoZ)
- [**Research Convo Guide**](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/payment-history/research/VHA-usability-study-9-2024/conversation-guide.md#conversation-guide-for-debt-resolution-team-vha-debt-portal-payment-history-september-2024)
- [**Research Plan**](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/payment-history/research/VHA-usability-study-9-2024/research-plan.md#research-plan-for-debt-resolution-team-vha-debt-portal-payment-history-september---october-2024)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: Maria will test using iOS + VoiceOver on Chrome.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** Maria
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:** September 26, 4:30pm ET
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/payment-history/research/VHA-usability-study-9-2024/conversation-guide.md) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](LINK)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot
- [x] **Review prototype** in `Codepen`
- [x] **Run pilot** with VFS Researcher
- [x] **Optional: Note Taking:** [Notes doc](https://docs.google.com/document/d/1g6h7RKXFqDZo8_b3f1Vk2qsmrgy0zhJhVvudUMdBLAs/edit)
- [x] Share findings with team from Pilot doc.
```
